000-vip-init.php

```diff
@@ -64,6 +70,27 @@
 	}
 }
 
+// Sites can be disabled if there is an overdue payment.
+// This constant is defined by VIP Go in config/wp-config.php.
+if ( Context::is_vip_env() && Context::is_overdue_locked() ) {
```

Comment:
Do we need any exceptions for logged-in VIP members? Could be a future enhancement I suppose. Just thinking of the case where someone on our end needs to do _something_ on a site while it's in lockout.

Folks that have the ability to sandbox sites can hack around it, just wondering if people think it's worthwhile to account for now.

Reply:
Good point! I have created AI-22 to track this improvement.